base/src/test/java/info/metadude/kotlin/library/c3media/ProductionApiTest.kt

```diff
@@ -214,6 +239,23 @@ class ProductionApiTest {
         }
     }
 
+    @Test
+    fun `Validates an event by guid response`() {
+        val call = service.getEvent(VALID_EVENT_GUID)
+        try {
+            val response = call.execute()
+            if (response.isSuccessful) {
+                val event = response.body()
+                assertThat(event!!).isNotNull()
```

Comment:
Comment "!!": I feel this is the wrong way around. Shouldn't this be like the following?

``` kotlin
assertThat(event).isNotNull
assertEvent(event!!)
```

or

``` kotlin
assertThat(event).isNotNull
event?.let(::assertEvent)
```